optimum/habana/transformers/models/llama/modeling_llama.py

```diff
@@ -58,10 +58,12 @@ def gaudi_llama_rmsnorm_forward(self, hidden_states):
         # mixed dtypes are not good for FusedRMSNorm, both inputs need to have same dtype
         if hidden_states.dtype != self.weight.dtype:
             orig_dtype = hidden_states.dtype
-            hidden_states = FusedRMSNorm.apply(hidden_states.to(self.weight.dtype), self.weight, self.variance_epsilon)
+            hidden_states = FusedRMSNorm.apply(
+                hidden_states.clone().to(self.weight.dtype), self.weight, self.variance_epsilon
```

Comment:
@sywangyi  why do we need clone here?

Reply:
we meet the similar issue with https://github.com/huggingface/optimum-habana/pull/507. I think. please see the history in 507

Reply:
remove the clone, since ppo disable rms in ppo.py